src/pages/OrganizationsPage.tsx

```diff
@@ -0,0 +1,188 @@
+      value: "Other",
+    },
+  ];
+
+  const handleChange = (value: string) => {
+    setQuery("");
+    setFilter(value);
+  };
+
+  const { loading, data } = useQuery(GetPartnerOrganizationsDocument, {
+    variables: {
+      facilityId: facility.id,
+      requestStatus: FacilityOrganizationStatus.Active,
+    },
+    onError: (err) =>
+      reportError(err, "OrganizationsPage/GetPartnerOrganizations"),
+  });
+
+  useEffect(
+    () =>
+      data &&
+      setFacilityOrgs(
+        data.facility.facilityOrganizations.filter(
+          (fo) =>
+            fo.organization.name.toLowerCase().includes(query.toLowerCase()) ||
+            fo.organization.type.toLowerCase() === query.toLowerCase()
+        )
+      ),
+    [data, query]
+  );
+
+  return (
+    <Content>
+      <Header
+        title="Professional Partners"
+        subtitle="View professional partner organizations and access their information."
+        extra={[
+          <Input.Group key="filter" compact>
+            <Select
+              defaultValue="name"
+              style={{ width: 120 }}
+              onChange={handleChange}
+            >
+              <Option value="name">Name</Option>
+              <Option value="type">Type</Option>
+            </Select>
+            {filter === "name" && (
+              <Input.Search
+                style={{ width: "auto" }}
+                placeholder={`Search by org name...`}
+                allowClear
+                value={query}
+                onChange={(event) => setQuery(event.target.value)}
+              />
+            )}
+            {filter === "type" && (
+              <AutoComplete
+                style={{ width: 200 }}
+                options={orgTypeOptions}
+                placeholder="Search by org type"
+                filterOption={(inputValue, option) =>
+                  option!.value
+                    .toUpperCase()
+                    .indexOf(inputValue.toUpperCase()) !== -1
+                }
+                onSelect={(value) => {
+                  setQuery(value);
+                }}
+              />
+            )}
+          </Input.Group>,
+        ]}
+      />
+      <div style={WRAPPER_STYLE}>
+        <Table dataSource={facilityOrgs} loading={loading}>
+          <Column
+            title=""
+            dataIndex={["organization", "logoImageUrl"]}
+            key="logoImageUrl"
+            render={(_text, facilityOrg: FacilityOrganization) => (
+              <>
+                <Avatar
+                  fallback={capitalize(facilityOrg.organization.name)}
+                  size={48}
+                />
+              </>
+            )}
+          />
+          <Column
+            title="Name"
+            dataIndex={["organization", "name"]}
+            key="organization.name"
+          />
+          <Column
+            title="Type"
+            dataIndex={["organization"]}
+            key="organization"
+            render={(org: Organization) => (
```

Comment:
i've started referring back to the dataSource's type:

```suggestion
            render={(org: FacilityOrganization['organization']) => (
```

or 

```suggestion
            render={(org: typeof facilityOrgs['organization']) => (
```

mostly because i like how it matches up with the `dataIndex` prop.

just an idea.